### PR TITLE
Documentation Updates

### PR DESCRIPTION
**Pull request by erikespinoza**
- Sample ZeroTrust Integrations
- Fix in Docker documentation
- Note and links to GitDB

### CODE REVIEW

File: docs/docker/README.md [hunk 1]
```diff
@@ -102,17 +114,6 @@ $ docker exec -it keymaster /usr/bin/htpasswd -B -c \
...
-
-By default a user is created. Let's start by deleting this and creating our own
-user.
-
-```
-$ rm -f ${KEYMASTER_DATA}/conf/passfile.htpass
-$ docker exec -it keymaster /usr/bin/htpasswd -B -c \
- /etc/keymaster/passfile.htpass $USERNAME
-```
-
```
Comment (cviecco) [inline docs/docker/README.md:115]: On the docker config... is the htpassword file not used?

File: docs/examples/zerotrust/huproxy.md [hunk 2]
```diff
@@ -0,0 +1,151 @@
+# HUProxy
+
+## What is HUProxy
+
+[HTTP(S)-Upgrade Proxy](https://github.com/google/huproxy) — Tunnel anything
+(but primarily SSH) over HTTP websockets.
```
Comment (cviecco) [inline docs/examples/zerotrust/huproxy.md:6]: This file in general is great... however I would not call it 'zerotrust', but vpnless. As you are actually fully trusting the proxy. I know that it is becoming common to mix those two, but for me zero trust also implies end to end ecryption.

**Review by cviecco — CHANGES_REQUESTED** [review]
Most of it is excellent, I only have one question and some suggestions for file naming.

File: docs/examples/zerotrust/huproxy.md (hunk 2 above)
Comment (erikespinoza) [reply docs/examples/zerotrust/huproxy.md]: Renamed per your request.

File: docs/docker/README.md (hunk 1 above)
Comment (erikespinoza) [reply docs/docker/README.md:115]: It is. I deleted this section because my original doc had a copy/paste error. This section is there twice.

**Review by erikespinoza — COMMENTED** [review]
Changes integrated.

**Review by cviecco — APPROVED** [review]

**Comment by cviecco** [comment]
thank yoi

**Pull request merged**